app/styles/modules/_branding.scss

```diff
@@ -62,3 +66,8 @@
     display: none;
   }
 }
+
+.chromeless #about-mozilla {
+  display: none;
+}
+
```

Comment:
Not sure if you want to relocate this `.chromeless` to be closer to the `.chromeless #fox-logo` on lines 39-41.

You could even probably do:
```scss
.chromeless {
  #fox-logo,
  #about-mozilla {
    display: none;
  }
}
```

Or nest the `.chromeless` declaration into the other selector as:
```scss
#about-mozilla {
  ...

  .chromeless & {
    display: none;
  }
}
```

Or leave it as-is since it's pretty damn readable.